kit/src/components/context_menu/context.js

```diff
@@ -1,37 +1,53 @@
+let click = (toggle) => function (ev) {
```

Comment:
i'm not familiar with this syntax. based off of [this](https://www.w3schools.com/Js/js_es6.asp#mark_arrow) it looks like the arrow can be used to make a lambda like this `(..) => {}` but I'm not familiar with `(..)  => function(..) {}`. is this valid javascript? where can I read more about it?

Reply:
its just a function creating a function.
so calling click(true) returns the function with toggle being true